client/assets/components/_dropdown.scss

```diff
@@ -1,7 +1,7 @@
 @import '../tools/functions.scss';
 
 .dropdown {
-  overflow: auto;
+  overflow: show;
```

Comment:
`show` doesn't seem to be a valid value for the overflow property. I am looking at [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow). Do you have a source for this?

Reply:
Thanks for pointing this out, I will change it to visible